pkg/k8s/k8s_pvc.go

```diff
@@ -9,17 +9,40 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func (c *Client) DeployPersistentVolumeClaim(ctx context.Context, name string, labels map[string]string, size resource.Quantity) error {
```

Comment:
We use `Create`, `Get`, and `Delete` for other resources. Should we use the same function naming for all resources to be more consistent?